### PR TITLE
Some error checking and user output notes

### PR DESCRIPTION
**Pull request by ghost**
Hi,

While running this it seems like the notebook hung but what really is happening is that notMNIST_large.tar.gz is taking a long time to extract. Also, since I looked at the contents of the directory a .DS_Store file was created.

### CODE REVIEW

**Comment by emmjaykay** [comment]
I signed it! ( the CLA)

File: tensorflow/examples/udacity/1_notmnist.ipynb [hunk 1]
```diff
@@ -184,10 +184,13 @@
         "\n",
         "def extract(filename):\n",
         "  tar = tarfile.open(filename)\n",
-        "  tar.extractall()\n",
-        "  tar.close()\n",
         "  root = os.path.splitext(os.path.splitext(filename)[0])[0]  # remove .tar.gz\n",
-        "  data_folders = [os.path.join(root, d) for d in sorted(os.listdir(root))]\n",
+        "  if not os.path.isdir(root):\n",
+        "    print \"Extracting data for %s. This may take a while. Please wait.\" % root\n",
```
Comment (vincentvanhoucke) [inline tensorflow/examples/udacity/1_notmnist.ipynb]: Would you mind using single quotes here for consistency?

**Comment by vincentvanhoucke** [comment]
LGTM @vrv

File: tensorflow/examples/udacity/1_notmnist.ipynb (hunk 1 above)
Comment (emmjaykay) [inline tensorflow/examples/udacity/1_notmnist.ipynb]: Sure, np! Thanks for looking at it

**Comment by vrv** [comment]
LG, squash the commits into one and we'll merge!

**Comment by emmjaykay** [comment]
Not sure how to do that with the GH interface. I made the changes in the web UI, not on git repo I checked out.

**Pull request closed without merging**